src/index.ts

```diff
@@ -2,16 +2,15 @@ import "reflect-metadata";
 import express from "express";
 import { userRouter } from "./routers/user.router";
 import { productRouter } from "./routers/product.router";
+import { invoiceRouter } from "./routers/invoice.router";
+import { invoiceDetailRouter } from "./routers/invoice-detail.router";
 
 const app: express.Application = express();
 
-app.use("/api/user", [
-  express.json(),
-  //express.urlencoded({ extended: false }),
-  userRouter,
-]);
+app.use("/api/user", [express.json(), userRouter]);
 app.use("/api/product", [express.json(), productRouter]);
-
+app.use("/api/invoice", [express.json(), invoiceRouter]);
```

Comment:
No, Why you have two API for the invoice ?
This is not the way.
I want you to have only one API. I did comment on this.

CLient send the invoice, complete! with the details.
the backend create the invoice.  (WITH A SQL TRANSACCTION)

Reply:
ok perfcet!